release notes/v0.44.0.md

```diff
@@ -0,0 +1,220 @@
+k6 v0.44.0 is here 🎉! This release includes:
+
+- A new experimental Web Crypto module.
+- A sampling option for the experimental tracing module.
+- Memory usage improvements.
+- Bug fixes and UX improvements.
+
+Some highlights from the browser module are:
+
+- `locator.click` is now asynchronous, which is a breaking change.
+- `browserContext.addCookies` has now been implemented.
+- `browserType.Connect` has been implemented so k6 can now connect to an already running Chrome/Chromium browser instance.
+- Web vitals are natively supported when working with the browser module.
+
+## Breaking changes
+
+- [xk6-browser#790](https://github.com/grafana/xk6-browser/pull/790) Converted `locator.click` to async to have feature parity with `page.click` and `elementHandle.click`. Users must remember to work with `promise.All` and `page.waitForNavigation()` when a click action results in navigation.
+
+    A `location.click` action that doesn't result in navigation can be used like so:
+    ```javascript
+    const tails = page.locator("input[value='Bet on tails!']");
+    await tails.click(),
+    ```
+
+    A `location.click` action that does result in a navigation can be used like so:
+    ```javascript
+    const tails = page.locator("input[value='Bet on tails!']");
+    await Promise.all([
+      page.waitForNavigation(),
+      tails.click(),
+    ]);
+    ```
+
+- [xk6-browser#817](https://github.com/grafana/xk6-browser/pull/817) We've removed `--no-sandbox` from the default Chrome launch arguments. Now Chrome will launch with a sandbox, which is a more secure way of running the browser. If you are running tests under a `root` user, the browser will no longer launch unless the `--no-sandbox` argument is supplied. You can still pass this flag when launching a new Chrome instance using the `args` parameter on `chromium.launch`:
+
+    ```javascript
+    const browser = chromium.launch({
+      args: ['no-sandbox'],
+    });
+    ```
+
+- [xk6-browser#844](https://github.com/grafana/xk6-browser/pull/844) Removed the exported `version` param from the root module. Users should from now on reference the k6 version instead of the browser module version.
+- [xk6-browser#838](https://github.com/grafana/xk6-browser/pull/838) Removed first meaningful paint metric. This metric is being deprecated across all the browsers, so we've removed it.
+
+## New features
+
+### A new experimental Web Crypto module [#2884](https://github.com/grafana/k6/pull/3007)
```

Comment:
```suggestion
### A new experimental Web Crypto API module [#2884](https://github.com/grafana/k6/pull/3007)
```
same

Reply:
Same comment as with the other related comment: if we rephrase, I'd rather rephrase to something along the lines of "A new experimental module implementing the [Web Crypto API](https://developer.mozilla.org/en-US/docs/Web/API/Web_Crypto_API) specification"

What do you think?

Reply:
Sounds good to me, except for placing the link to the headline. I will adjust accordingly :+1:

Reply:
Please find the changes there https://github.com/grafana/k6/compare/39ffea69ba60d46dff1ae65ac7101404a3d6f64b..76f3211020e2136f9dcf0771b2ef5e1039edb1da